Apps/Sandcastle/gallery/development/3D Models Outline.html

```diff
@@ -0,0 +1,184 @@
+    <script type="text/javascript" src="../../../ThirdParty/requirejs-2.1.20/require.js"></script>
+    <script type="text/javascript">
+    require.config({
+        baseUrl : '../../../Source',
+        waitSeconds : 60
+    });
+    </script>
+</head>
+<body class="sandcastle-loading" data-sandcastle-bucket="bucket-requirejs.html">
+<style>
+    @import url(../templates/bucket.css);
+    #toolbar {
+        background: rgba(42, 42, 42, 0.8);
+        padding: 4px;
+        border-radius: 4px;
+    }
+    #toolbar input {
+        vertical-align: middle;
+        padding-top: 2px;
+        padding-bottom: 2px;
+    }
+</style>
+<div id="cesiumContainer" class="fullSize"></div>
+<div id="loadingOverlay"><h1>Loading...</h1></div>
+<div id="toolbar">
+    <table>
+    <tbody>
+    <tr>
+        <td>Size</td>
+        <td>
+            <input type="range" min="0" max="20" step="1" data-bind="value: highlightSize, valueUpdate: 'input'">
+            <input type="text" size="5" data-bind="value: highlightSize">
+        </td>
+    </tr>
+    <tr>
+        <td>Opacity</td>
+        <td>
+            <input type="range" min="0" max="100" step="1" data-bind="value: highlightOpacity, valueUpdate: 'input'">
+            <input type="text" size="5" data-bind="value: highlightOpacity">
+        </td>
+    </tr>
+
+</tbody></table>
+</div>
+<script id="cesium_sandcastle_script">
+function startup(Cesium) {
+    'use strict';
+//Sandcastle_Begin
+var viewer = new Cesium.Viewer('cesiumContainer', {
+    infoBox : false,
+    selectionIndicator : false,
+    shadows : true,
+    // Explicitly request a stencil buffer so highlighting will work.
+    contextOptions: {
+        webgl : {
+            stencil: true
+        }
+    }
+});
+
+var entity = null;
+
+// The viewModel tracks the state of our mini application.
+var viewModel = {
+    highlight: true,
+    highlightSize: 2.0,
+    highlightOpacity: 100.0,
+    highlightColor: new Cesium.Color(1.0, 1.0, 1.0, 1.0)
+};
+
+// Convert the viewModel members into knockout observables.
+Cesium.knockout.track(viewModel);
+
+// Bind the viewModel to the DOM elements of the UI that call for it.
+var toolbar = document.getElementById('toolbar');
+Cesium.knockout.applyBindings(viewModel, toolbar);
+
+
+function createModel(url, height) {
+    viewer.entities.removeAll();
+
+    var position = Cesium.Cartesian3.fromDegrees(-123.0744619, 44.0503706, height);
+    var heading = Cesium.Math.toRadians(135);
+    var pitch = 0;
+    var roll = 0;
+    var orientation = Cesium.Transforms.headingPitchRollQuaternion(position, heading, pitch, roll);
+
+    entity = viewer.entities.add({
+        name : url,
+        position : position,
+        orientation : orientation,
+        model : {
+            uri : url,
+            minimumPixelSize : 128,
+            maximumScale : 20000,
+            highlight: viewModel.highlight,
+            highlightSize: viewModel.highlightSize,
```

Comment:
Throughout, we should consider calling this "silhouette", instead of "highlight", since "hightlight" could mean many things like #4547 or highlighting 3D Tiles features.